### PR TITLE
replaced new lines with System.lineSeparator()

### PR DESCRIPTION
**Pull request by mareknovotny**
this showed up in https://github.com/windup/windup/pull/1194, so to clear that all in once

### CODE REVIEW

File: config-xml/addon/src/main/java/org/jboss/windup/config/parser/xml/PhaseHandler.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public Void processElement(ParserContext context, Element element) throws Config
         Class<? extends RulePhase> phase = phaseFinder.findPhase(phaseStr);
         if (phase == null)
         {
-            String phasesListing = phaseFinder.getAvailablePhases().stream().map(clazz -> clazz.getSimpleName()).collect(Collectors.joining("\n    "));
```
Comment (jsight) [inline config-xml/addon/src/main/java/org/jboss/windup/config/parser/xml/PhaseHandler.java]: It looks like this one changes it without keeping the additional spaces. Is that intentional?

**Comment by jsight** [comment]
I just had one comment. I also noticed that some changed "\n\t" into "\n \t" (extra space), but I'm not sure if that important.

File: config-xml/addon/src/main/java/org/jboss/windup/config/parser/xml/PhaseHandler.java (hunk 1 above)
Comment (mareknovotny) [reply config-xml/addon/src/main/java/org/jboss/windup/config/parser/xml/PhaseHandler.java]: probably not. I will fix it just for sure.

**Comment by mareknovotny** [comment]
retest

**Comment by mareknovotny** [comment]
Random test failures again due furnace container not able to load/unload something :/.

**Comment by mareknovotny** [comment]
retest

File: test-files/soa/soa5_gateways_routers/src/org/jboss/soa/esb/samples/quickstart/helloworld/MyJMSListenerAction.java [hunk 2]
```diff
@@ -50,7 +50,7 @@ public Message playWithMessage(Message message) throws Exception {
     }
 
     private void logHeader() {
-        System.out.println("\n&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&");
+        System.out.println(System.lineSeparator() &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&");
     }
```
Comment (OndraZizka) [inline test-files/soa/soa5_gateways_routers/src/org/jboss/soa/esb/samples/quickstart/helloworld/MyJMSListenerAction.java]: Huh?

File: graph/api/src/main/java/org/jboss/windup/graph/GraphUtil.java [hunk 3]
```diff
@@ -29,49 +29,49 @@ private static final void vertexAsString(Vertex vertex, int depth, String withEd
...
-            sb.append("\n    ").append(indent).append(propKey).append(": ").append("" + vertex.getProperty(propKey));
+            sb.append(System.lineSeparator()).append(indent).append(propKey).append(": ").append("" + vertex.getProperty(propKey));
         }
 
         if (withEdgesOfLabel == null || depth == 0)
         {
             if (hasProps)
-                sb.append("\n    ").append(indent);
+                sb.append(System.lineSeparator()).append(indent);
             if (hasEdges)
```
Comment (OndraZizka) [inline graph/api/src/main/java/org/jboss/windup/graph/GraphUtil.java]: You have omitted the spaces, making the log less radable.

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderApplicationPieChartDirective.java [hunk 4]
```diff
@@ -103,31 +103,31 @@ private void drawPie(Writer writer, Map<String, Integer> data, String elementID)
...
         writer.append("<script type='text/javascript'>");
-        writer.append("\nWINDUP_PACKAGE_PIE_DATA = typeof(WINDUP_PACKAGE_PIE_DATA) == 'undefined' ? {} : WINDUP_PACKAGE_PIE_DATA;");
-        writer.append("\nWINDUP_PACKAGE_PIE_DATA['").append(elementID).append("'] = [];");
+        writer.append(System.lineSeparator()+" WINDUP_PACKAGE_PIE_DATA = typeof(WINDUP_PACKAGE_PIE_DATA) == 'undefined' ? {} : WINDUP_PACKAGE_PIE_DATA;");
+        writer.append(System.lineSeparator()+" WINDUP_PACKAGE_PIE_DATA['").append(elementID).append("'] = [];");
         for (PieSort p : pieList)
         {
-            writer.append("\nWINDUP_PACKAGE_PIE_DATA['").append(elementID).append("'].push({label: '" + p.label + "', data: ")
+            writer.append(System.lineSeparator()+" WINDUP_PACKAGE_PIE_DATA['").append(elementID).append("'].push({label: '" + p.label + "', data: ")
                         .append(p.value.toString()).append("});");
```
Comment (OndraZizka) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderApplicationPieChartDirective.java]: Why the extra space here? Omissions like this make me wonder how many bugs were introduced.

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderApplicationPieChartDirective.java [hunk 5]
```diff
@@ -103,31 +103,31 @@ private void drawPie(Writer writer, Map<String, Integer> data, String elementID)
...
+                      System.lineSeparator()+"           var len = " + dataVarName + ".length;" +
+                      System.lineSeparator()+"           return jQuery.Color({" +
+                      System.lineSeparator()+"               hue: ((index*0.95*360/len) + 90/len) % 360," +
+                      System.lineSeparator()+"               saturation: 0.95," +
+                      System.lineSeparator()+"               lightness: ((index%4 == 3 ? 1:0)/-4)+0.55, alpha: 1" +
+                      System.lineSeparator()+"           }).toHexString();" +
+                      System.lineSeparator()+"       })");
+        writer.append(System.lineSeparator()+"   });");
+        writer.append(System.lineSeparator()+" });");
         writer.append("</script>");
```
Comment (OndraZizka) [inline rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderApplicationPieChartDirective.java]: Now this is really crazy. These files do not need to have platform specific line endings at all. This has improved nothing at the cost of making the code ugly.

File: tests/src/test/java/org/jboss/windup/tests/bootstrap/AbstractBootstrapTestWithRules.java [hunk 6]
```diff
@@ -27,7 +27,7 @@
             + "        <targetTechnology id=\"eap7\" versionRange=\"[7,)\" />\n"
             + "        <tag>test-tag-eap</tag>\n"
             + "    </metadata>\n"
-            + "\n"
+            + System.lineSeparator()
             + "    <rules>\n"
```
Comment (OndraZizka) [inline tests/src/test/java/org/jboss/windup/tests/bootstrap/AbstractBootstrapTestWithRules.java]: I wonder why are you changing one \n while leaving the others?

File: ui/tests/src/test/java/org/jboss/windup/addon/ui/WindupUpdateDistributionCommandTest.java [hunk 7]
```diff
@@ -260,7 +260,7 @@ private void changeUiAddonVersion(File addonsDir, String currentUiVersion)
                     "org-jboss-windup-ui-windup-ui-" + WINDUP_OLD_VERSION.replaceAll("\\.", "-"));
         olderVersionAddonDir.mkdirs();
 
-        log.warning("Replacing the addon: \n  " + currentAddonDir + "\n  " + olderVersionAddonDir);
+        log.warning("Replacing the addon: \n  " + currentAddonDir + System.lineSeparator() + olderVersionAddonDir);
         try
```
Comment (OndraZizka) [inline ui/tests/src/test/java/org/jboss/windup/addon/ui/WindupUpdateDistributionCommandTest.java]: Missing the space

File: test-files/soa/soa5_gateways_routers/src/org/jboss/soa/esb/samples/quickstart/helloworld/MyJMSListenerAction.java (hunk 2 above)
Comment (mareknovotny) [reply test-files/soa/soa5_gateways_routers/src/org/jboss/soa/esb/samples/quickstart/helloworld/MyJMSListenerAction.java]: are you scary from test source file? I guess it was taken from some example - not really matter if bad or good ;)

File: graph/api/src/main/java/org/jboss/windup/graph/GraphUtil.java (hunk 3 above)
Comment (mareknovotny) [reply graph/api/src/main/java/org/jboss/windup/graph/GraphUtil.java]: new line with tab the result will look very the same IMO

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderApplicationPieChartDirective.java (hunk 4 above)
Comment (mareknovotny) [reply rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderApplicationPieChartDirective.java]: I did mass find and replace, and in some cases it doesn't matter if there is additional or missing space(s). This code should generate javascript variable with content ;)

File: rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderApplicationPieChartDirective.java (hunk 5 above)
Comment (mareknovotny) [reply rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderApplicationPieChartDirective.java]: but keeping `\n` when everywhere it is replaced doesn't seem ok

File: tests/src/test/java/org/jboss/windup/tests/bootstrap/AbstractBootstrapTestWithRules.java (hunk 6 above)
Comment (mareknovotny) [reply tests/src/test/java/org/jboss/windup/tests/bootstrap/AbstractBootstrapTestWithRules.java]: no big reason, just not catch them, but this is test rules, so no big deal

**Pull request merged**